stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -883,53 +600,30 @@ extension ${Self}: BinaryFloatingPoint {
           clamped -= greatestFiniteExponent
         }
       }
-      let scale = ${Self}(sign: .plus,
+      let scale = ${Self}(
+        sign: .plus,
         exponentBitPattern: UInt(Int(${Self}._exponentBias) + clamped),
-        significandBitPattern: 0)
+        significandBitPattern: 0
+      )
       result = result * scale
     }
     self = result
   }
 
-  /// Creates a NaN ("not a number") value with the specified payload.
```

Comment:
This isn't a protocol requirement, so we can't delete the documentation here.

Reply:
Restored.